website/urls.py

```diff
@@ -12,6 +13,9 @@
     # Cite Page for publications
     url(r'^cite/$', views.cite, name='cite'),
 
+    # News Post display page
+    url(r'^news/(?P<news_id>.*?)/$', views.news_page, name='news_page'),
```

Comment:
Actually this is required by the share buttons. When we share the news we are actually sharing a dedicated page with the news and that page has the required meta tags to render the shared post with proper title, description etc. So pressing the share button on a news is same as pasting the link dipy.herokuapp.com/news/3 in out google plus feed.